### DISCUSSION TITLE
[Bug] The default oap startup in .env under the docker directory fails

### DISCUSSION THREAD

**Discussion by jiewang-2023** [apache/skywalking#12516] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

Started by docker compose, failed to start normally, here is the startup command, docker-compose --profile banyandb up -d
![image](https://github.com/user-attachments/assets/03f42944-5b76-4061-a258-5621c55df742)

### What you expected to happen

Start normally through docker compose, and give examples of mounting configuration files and data stores outside the disk

### How to reproduce

Started by docker-compose file, the configuration has not been modified

### Are you willing to submit a pull request to fix on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Follow quick start doc. Don't report bugs as you don't follow our guidelines.

**Comment by jiewang-2023** [comment]
Use the script in the document to start normally https://skywalking.apache.org/docs/main/v10.0.1/en/setup/backend/backend-docker/

Use docker directly The mirror in .env under the directory failed to start BANYANDB_IMAGE=ghcr.io/apache/skywalking-banyandb:84f32b3969cdcc676aaee428383b34b3b67dbdf5
OAP_IMAGE=ghcr.io/apache/skywalking/oap:latest
UI_IMAGE=ghcr.io/apache/skywalking/ui:latest
Changed to
BANYANDB_IMAGE=apache/skywalking-banyandb:0.6.1
OAP_IMAGE=apache/skywalking-oap-server:10.0.1
UI_IMAGE=apache/skywalking-ui:10.0.1
Can start normally,
Start command: docker-compose --profile banyandb up -d
Please check again if there is any problem.

**Comment by wu-sheng** [comment]
The showcase has matched version up and running.
I don't know how you do with random versions.

oap 10.0.1 should be with BanyanDB 0.6.1
Dev versions(latest) is more about you knew what you are going to do. Showcase has versions listed in.

**Reply by jiewang-2023** [reply]
OK, thanks
